app/use/middlewares.js

```diff
@@ -4,11 +4,7 @@ let state = config.middlewares
 module.exports = function () {
   return {
     append (mw) {
-      state.push(mw)
-    },
-
-    set (mws) {
-      state = [...mws]
+      state = [...state, ...mw]
```

Comment:
This is definitely not the same behavior...

Reply:
I think I understood the initial behaviour way after I did that change... Let's talk that topic around a cup of chocky, shall we ?